plugins/inventory/vmware_vm_inventory.py

```diff
@@ -554,6 +552,23 @@ def build_containers(containers, vim_type, names, filters):
         return []
 
 
+def dict_merge(a, b):
```

Comment:
ansible-core already provides an `dict_merge` function `lib/ansible/module_utils/common/dict_transformations.py`. Can you please us it instead?

`from ansible.module_utils.common.dict_transformations import dict_merge`

Reply:
@Akasurde mentioned that too. Maybe it is better to rename it to avoid the confusion; e.g. `dict_in_place_merge` or something else.

https://github.com/ansible-collections/community.vmware/pull/441#discussion_r504661200
> The purpose of `dict_transformations.dict_merge` is to merge `a` dict and `b` dict in a new one `c` dict. For our usage we need to merge `b` into `a` with zero-copy. `deepcopy` is very expensive call and have no added value for our case. And from the profile I did, it was taking 50% of process time( see below `cumtime`).
> 
> **main branch**
> 
> ```
>          8979909 function calls (8562267 primitive calls) in 6.465 seconds
> 
>    Ordered by: internal time
>    List reduced from 3757 to 1 due to restriction <'to_nested_dict'>
> 
>    ncalls  tottime  percall  cumtime  percall filename:lineno(function)
>        82    0.004    0.000    0.523    0.006 vmware_vm_inventory.py:572(to_nested_dict)
> ```
> 
> (#441 branch)
> 
> ```
>          35063500 function calls (29865325 primitive calls) in 18.657 seconds
> 
>    Ordered by: internal time
>    List reduced from 3773 to 1 due to restriction <'to_nested_dict'>
> 
>    ncalls  tottime  percall  cumtime  percall filename:lineno(function)
>        82    0.066    0.001    9.201    0.112 /Users/dc/scratch/test_ansible_pr/collections/ansible_collections/community/vmware/plugins/inventory/vmware_vm_inventory.py:557(to_nested_dict)
> ```